### PR TITLE
Display personal schedule

### PR DESCRIPTION
**Pull request by mikeverf**
> Add a personal schedule button that when clicked opens a fullscreen modal from the bottom. In it, there is a flatList with the subscribed tracks of 1 particular user (now id '10' from dummy-data).
I also cleaned up the components folder and gave them better names

This PR closes ticket #9

### CODE REVIEW

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-9-display_my_personal_schedule

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

File: components/closePersonalScheduleButton.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  close: {
+    tintColor: COLORS.white,
+    width: METRICS.gridSize * 3,
+    height: METRICS.gridSize * 3,
+  },
+});
+
+const ClosePersonalScheduleButton = ({ goBack }) => (
+  <TouchableWithoutFeedback onPress={() => goBack()} >
```
Comment (thibmaek) [inline components/closePersonalScheduleButton.js]: You can omit the bind here: `onPress={goBack}`

File: components/personalScheduleButton.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+const PersonalScheduleButton = ({ onPress }) => (
+  <TouchableWithoutFeedback onPress={!DISABLED ? onPress : null}>
+    <View style={[
+      styles.buttonContainer,
+      IS_IPHONE_X ? styles.iPhoneXPaddingBottom : null,
+      DISABLED ? styles.disabled : null,
+    ]}
+    >
+      <Text style={styles.buttonText}>{'My Schedule'}</Text>
```
Comment (thibmaek) [inline components/personalScheduleButton.js]: JSX expression is not needed here: `{'My Schedule'}` vs `My Schedule`

File: screens/home/HomeContainer.js [hunk 3]
```diff
@@ -21,27 +30,32 @@ export default class HomeContainer extends Component {
...
+                index={index}
+                total={total}
+                tracks={tracks}
+                onNextTap={this.handleTouchableTap}
+              />
+            )
+          }
+        >
+          {
+            tracks.map((track, i) => <HomeScreen key={track.id} trackName={track.title} trackId={track.id} sessions={this.trackSessions[i]} />)
```
Comment (thibmaek) [inline screens/home/HomeContainer.js]: Hmmm can you use something other than the mapping index `i` here? Perhaps you could use the `id` to get the data you need?

File: screens/personalSchedule/PersonalScheduleScreen.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  flatlist: {
+    paddingHorizontal: METRICS.gridSize * 2,
+    paddingBottom: CONSTANTS.paddingBottom,
+  },
+});
+
+const PersonalScheduleScreen = ({ sessions, navigation }) => (
+  <View style={styles.container}>
+    <View style={styles.header}>
+      <Text style={styles.title}>{'My Schedule'}</Text>
```
Comment (thibmaek) [inline screens/personalSchedule/PersonalScheduleScreen.js]: Same remark as above here, jsx expression not needed.

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-9-display_my_personal_schedule

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-9-display_my_personal_schedule

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

File: screens/home/HomeContainer.js [hunk 5]
```diff
@@ -1,47 +1,70 @@
...
   get trackSessions() {
     return tracks.map((track) => {
       const trackSessions = sessions.filter(session => (
         session.tags.some(sessionTrack => sessionTrack.id === track.id)
       ));
-      return sortByDate(trackSessions);
+      const sortedTrackSessions = sortByDate(trackSessions);
+      return {
+        id: track.id,
+        tracks: [...sortedTrackSessions],
```
Comment (thibmaek) [inline screens/home/HomeContainer.js]: Can be removed here since `sortByDate` will return a shallow copy by default.

**Pull request merged**